src/components/colorPicker/ColorSwatch.js

```diff
@@ -143,6 +143,7 @@ export default class ColorSwatch extends PureBaseComponent {
         onPress={this.onPress}
         style={[this.styles.container, style]}
         onLayout={this.onLayout}
+        accessibilityLabel={ColorName.name(color)[1]}
```

Comment:
can we simplify the functionality so it will return the color name instead of an array. 
I think the array is a little confusing as a return value, and since we're exporting this functionality I think it will be more clear if it would just return a string name

Reply:
The thing is that the color returned is not necessarily the original color so I think it's important to return the hex as well. The last value is an indication if the original color is part of the map and will allow adding colors to the map when we'll want this functionality in the future. If we go to the mask this in our Colors class we can return only the name string.

Reply:
Moved example to use `Colors.getColorName(color)`